docs/developer-guide/new-environment.rst

```diff
@@ -0,0 +1,141 @@
+################################
+Deploying into a new environment
+################################
+
+Deploying Sasquatch into a new environment requires multiple ArgoCD syncs with some manual information gathering and updating in between.
+
+
+Enable Sasquatch in Phalanx
+===========================
+
+#. Cut a `Phalanx`_ development branch.
+#. Ensure the ``strimzi`` and ``strimzi-access-operator`` Phalanx applications are enabled and synced in the new environment by adding them to the :samp:`environments/values-{environment}.yaml` file, and adding a blank :samp:`values-{environment}.yaml` file to their ``applications/`` directories.
+   `These docs <https://phalanx.lsst.io/developers/switch-environment-to-branch.html>`_ can help you enable them from your development branch.
+#. Enable the ``sasquatch`` app in the environment.
+   For the :samp:`applications/sasquatch/values-{environment}.yaml` file, copy one from an existing environment that has the same enabled services that you want in the new environment.
+   Change all of the environment references to the new environment, and change or add anything else you need for the new environment.
+#. Comment out any ``loadBalancerIP`` entries in the :samp:`applications/sasquatch/values-{environment}.yaml` file.
+   We'll fill these in later.
+#. In the new environment's ArgoCD, point the ``sasquatch`` app at your Phalanx development branch, and sync it.
+
+This first sync will not be successful.
+The `cert-manager`_ ``Certificate`` resource will be stuck in a progressing state until we update some values and provision some DNS.
+
+.. _Phalanx: https://phalanx.lsst.io
+.. _cert-manager: https://cert-manager.io/
+
+Gather IP addresses and update Phalanx config
+=============================================
+
+.. note::
+
+   The public IP address gathering and modification described here only applies to environments deployed on `GCP`_.
+   This process will be different for other types of environments.
+
+#. Get the node ids of the kafka brokers.
+   In this example, the node ids are ``3``, ``4``, and ``5``:
```

Comment:
I would use "broker IDs" to make it clear we are talking about the brokers and use 0, 1, and 2 to be consistent with the output and ``externalListener`` example configuration below.